baseplate/lib/experiments/__init__.py

```diff
@@ -81,51 +97,59 @@ def __init__(
         config_watcher: FileWatcher,
         server_span: Span,
         context_name: str,
+        cfg_data: Optional[Dict[str, Dict[str, str]]] = None,
+        global_cache: Optional[Dict[str, Optional[Experiment]]] = None,
         event_logger: Optional[EventLogger] = None,
     ):
         self._config_watcher = config_watcher
         self._span = server_span
         self._context_name = context_name
         self._already_bucketed: Set[str] = set()
-        self._experiment_cache: Dict[str, Optional[Experiment]] = {}
+        self._cfg_data = cfg_data
+        self._global_cache = global_cache if global_cache is not None else {}
         if event_logger:
             self._event_logger = event_logger
         else:
             self._event_logger = DebugLogger()
 
-    def _get_config(self, name: str) -> Optional[Dict[str, str]]:
+    def _get_config(self) -> Dict[str, Dict[str, str]]:
+        """Warn: Deprecate in Baseplate 2.0."""
```

Comment:
```suggestion
        # Warn: Deprecate in Baseplate 2.0.
```

since this is more of an internal comment, and not something we'd want to show in public documentation, it should just be a comment rather than a docstring